skbio/io/format/genbank.py

```diff
@@ -0,0 +1,865 @@
+
+Now we can read it as ``DNA`` object:
+
+>>> from skbio import DNA, RNA, Sequence
+>>> dna_seq = DNA.read(gb)
+>>> dna_seq
+DNA
+-----------------------------------------------------------------
+Metadata:
+    'ACCESSION': '3K1V_A'
+    'COMMENT': 'SEQRES.'
+    'DEFINITION': 'Chain A, Structure Of A Mutant Class-I Preq1.'
+    'FEATURES': <class 'list'>
+    'KEYWORDS': '.'
+    'LOCUS': <class 'dict'>
+    'REFERENCE': <class 'list'>
+    'SOURCE': <class 'dict'>
+    'VERSION': '3K1V_A  GI:260656459'
+Positional metadata:
+    0: <dtype: bool>
+Stats:
+    length: 34
+    has gaps: False
+    has degenerates: False
+    has non-degenerates: True
+    GC-content: 35.29%
+-----------------------------------------------------------------
+0 AGAGGTTCTA GCACATCCCT CTATAAAAAA CTAA
+
+
+Since this is a riboswitch molecule, we may want to read it as ``RNA``.
+As the GenBank file usually have ``'t'`` instead of ``'u'`` in the
+sequence, we can not read it as ``RNA`` directly without conversion.
+We can read it as ``DNA`` and then ``transcribe`` it into ``RNA``:
+
+>>> rna_seq = dna_seq.transcribe()
+>>> rna_seq
+RNA
+-----------------------------------------------------------------
+Metadata:
+    'ACCESSION': '3K1V_A'
+    'COMMENT': 'SEQRES.'
+    'DEFINITION': 'Chain A, Structure Of A Mutant Class-I Preq1.'
+    'FEATURES': <class 'list'>
+    'KEYWORDS': '.'
+    'LOCUS': <class 'dict'>
+    'REFERENCE': <class 'list'>
+    'SOURCE': <class 'dict'>
+    'VERSION': '3K1V_A  GI:260656459'
+Positional metadata:
+    0: <dtype: bool>
+Stats:
+    length: 34
+    has gaps: False
+    has degenerates: False
+    has non-degenerates: True
+    GC-content: 35.29%
+-----------------------------------------------------------------
+0 AGAGGUUCUA GCACAUCCCU CUAUAAAAAA CUAA
+
+
+Or we can convert ``'t'`` to ``'u'`` in the sequence and then read it
+as ``RNA``:
+
+>>> gb_rna = gb
+>>> gb_rna[-2] = gb_rna[-2].replace('t', 'u')
+>>> rna_seq_2 = RNA.read(gb_rna)
+>>> rna_seq_2 == rna_seq
+True
+
+>>> from io import StringIO
+>>> with StringIO() as fh:
+...     print(dna_seq.write(fh, format='genbank').getvalue())
+LOCUS       3K1V_A   34 bp   RNA   linear   SYN   10-OCT-2012
+DEFINITION  Chain A, Structure Of A Mutant Class-I Preq1.
+ACCESSION   3K1V_A
+VERSION     3K1V_A  GI:260656459
+KEYWORDS    .
+SOURCE      synthetic construct
+  ORGANISM  synthetic construct
+            other sequences; artificial sequences.
+REFERENCE   1  (bases 1 to 34)
+  AUTHORS   Klein,D.J., Edwards,T.E. and Ferre-D'Amare,A.R.
+  TITLE     Cocrystal structure of a class I preQ1 riboswitch
+  JOURNAL   Nat. Struct. Mol. Biol. 16 (3), 343-344 (2009)
+  PUBMED    19234468
+COMMENT     SEQRES.
+FEATURES             Location/Qualifiers
+       source        1..34
+                     /db_xref="taxon:32630"
+                     /mol_type="other RNA"
+                     /organism="synthetic construct"
+ORIGIN
+        1 agaggttcta gcacatccct ctataaaaaa ctaa
+//
+<BLANKLINE>
+
```

Comment:
I think we should have an example of writing out `RNA` and seeing the `U`s of `RNA` turn into `t` for GenBank